cpp/src/utilities/spmv_1D.cu

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "spmv_1D.cuh"
+
+namespace cugraph {
+namespace opg {
+template <typename VT, typename ET, typename WT>
+OPGcsrmv<VT, ET, WT>::OPGcsrmv(const raft::comms::comms_t &comm_,
+                               VT *local_vertices_,
+                               VT *part_off_,
+                               ET *off_,
+                               VT *ind_,
+                               WT *val_,
+                               WT *x)
+  : comm(comm_),
+    local_vertices(local_vertices_),
+    part_off(part_off_),
+    off(off_),
+    ind(ind_),
+    val(val_)
+{
+  stream = nullptr;

```

Comment:
Shouldn't we take stream from raft handle?

Reply:
It is just a place holder currently. But you are correct it should come from the raft handle if cuGraph was really supporting streams. We can definitely look at stream support and priorities at the library level in another issue.

Reply:
https://github.com/rapidsai/cugraph/issues/957

Reply:
Yeah... but should at least be consistent in handling stream.

stream is defined as the member variable of PageRank class below
https://github.com/rapidsai/cugraph/pull/947/files#diff-1ca4da61cab77812b7b10fe0adac80f9R41

and here stream is defined here as a local variable.

Some kernels specify stream and some kernels do not specify stream.

Fixing this really requires reading and analyzing the whole code again to properly use stream from RAFT handle. If we are consistent, just searching for stream, and replacing stream with handle.get_stream() will fix the problem.

If we are not ready to take RAFT handle as input parameter, should we at least be consistent in exposing stream?